Makefile

```diff
@@ -37,9 +37,9 @@ build:
 
 .EXPORT_ALL_VARIABLES:
 help:
-	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" ${ORG}/${NAME} -h
+	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" -v "$(PWD)/templates:${WORKDIR}/templates" ${ORG}/${NAME} -h
 
 .EXPORT_ALL_VARIABLES:
 run:
-	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" ${ORG}/${NAME} ${FLAGS}
-	python -m webbrowser -t "file://$(PWD)/render/index.html"
\ No newline at end of file
+	docker run --env-file .env -it --rm -v "$(PWD)/render:${WORKDIR}/render" -v "$(PWD)/templates:${WORKDIR}/templates" ${ORG}/${NAME} ${FLAGS}
```

Comment:
I'm somewhat inclined against this. I understand the hassle of rebuilding the image while developing a them, but I kindof think the run artifact should be as fully packages as possible. What about creating some sort of dev make target for this use case?